reframe/frontend/cli.py

```diff
@@ -84,6 +84,9 @@ def main():
         help='Load checks recursively')
 
     # Select options
+    select_options.add_argument(
+        '--try-module', action='append', dest='module_map', default=[],
+        help='Map a module name to a different one')
```

Comment:
This is not the option described in #76.

Reply:
It should accept a mapping file and should not append its values. Perhaps we should call it `--module-file`.